src/lego_robot/main.c

```diff
@@ -68,7 +68,7 @@ void follow_black_line(int motor_a, int motor_b)
 void drive_to_start()
 {
     enable_servo(0);
-    msleep(10);
+    msleep(30);
```

Comment:
Why is it important to increase the delay?

Reply:
Fortnite